GVFS/GVFS.Common/QueryGVFSVersionFromOrgInfo.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Net.Http;
+
+namespace GVFS.Common
+{
+    public class QueryGVFSVersionFromOrgInfo : IQueryGVFSVersion

```

Comment:
I think this might be related to @mjcheetham's comment in OrgInfoApiClient.cs.

I am having trouble understanding why we have both `QueryGVFSVersionFromOrgInfo` and `OrgInfoApiClient`.  As best I can tell all this class is doing is passing through a call to `QueryNewestVersion` (saving the caller from having to pass in the values it used when it constructed the `QueryGVFSVersionFromOrgInfo`).

Is it possible to reduce the level of abstractions here?